diagnostic.md

```diff
@@ -7,36 +7,46 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git@github.com:yenpraja610/git-basics-diagnostic.git -->
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+<!-- move to the folder
+git branch
+git checkout
+atom <file>
+-->
```

Comment:
The answer that we're looking for is creating a new branch to make your changes in, but no points will be taken off since your intention is understood.